wave_vision/src/fast_detector.cpp

```diff
@@ -0,0 +1,87 @@
+// Libwave Headers
+#include "wave/vision/fast_detector.hpp"
+
+namespace wave {
+
+FASTDetector::FASTDetector(const std::string &config_path) {
+    // Extract parameters from .yaml file.
+    ConfigParser parser;
```

Comment:
I am skeptical that a "Feature Detector" class should contain a file parser. They are separate concerns.

Certainly a configuration file should not be the only way to initialize a feature detector.